docs/reference/migration/migrate_7_0/api.asciidoc

```diff
@@ -22,9 +22,76 @@ while handling a write operation, it may expose a version that will then be reus
 new primary.
 
 Due to that issue, internal versioning can no longer be used and is replaced by a new
-method based on sequence numbers. See <<optimistic-concurrency-control>> for more details.
-
-Note that the `external` versioning type is still fully supported.
+method based on <<optimistic-concurrency-control,sequence numbers>>. To switch
+to the new method, follow these steps:
+
+. Perform a <<rolling-upgrades,rolling upgrade>> to 6.8.
+
+. <<reindex-upgrade,Reindex>> any indices created before 6.0. This ensures
+documents in those indices have sequence numbers.
++
+To get the {es} version in which an index was created, use the
+<<indices-get-index,get index API>> with the `human` parameter:
++
+--
+.*API Example*
+[%collapsible]
+====
+[source,console]
+----
+GET /*?human
+----
+// TEST[setup:my_index]
+
+The response returns a `settings.index.version.created_string` property for
+each index:
+
+[source,console-response]
+----
+{
+  "my-index-000001": {
+    ...
+    "settings": {
+      "index": {
+        "creation_date_string": "2099-01-01T00:00:00.000Z",
+        "routing": {
+          "allocation": {
+            "include": {
+              "_tier_preference": "data_content"
+            }
+          }
+        },
+        "number_of_shards": "1",
+        "provided_name": "my-index-000001",
+        "creation_date": "4070908800",
+        "number_of_replicas": "1",
+        "uuid": "c89-MHh8RwKwS1r7Turg2g",
+        "version": {
+          "created_string": "6.0.0",           <1>
```

Comment:
nit: I wonder if it was a bit clearer what to look for if the response stated that the index was created in a 5.x version?

Reply:
:doh: Yes. That would be much clearer. I'll get that fixed up. Thanks for catching that @henningandersen.